src/OpenTelemetry.Extensions.Hosting/README.md

```diff
@@ -21,8 +21,6 @@ and metrics (`MeterProvider`) in [ASP.NET
 
 ## Extension method reference
 
-### Current OpenTelemetry SDK v1.4.0 and newer extensions
```

Comment:
consider a link to the example app, which shows this in action.

Reply:
There is a usage section below just not shown on the diff unless you expand it

Reply:
Or if you are saying in addition to that a link would be good, find by me 😄

Reply:
yea on top of the basic usage shown here.

Reply:
the example, if followed correctly, will help avoid mistakes like adding ActivitySource to DI etc. So good to link from this